sphinx_doc/howtos/tunning/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _docs_tunning:
+
+Tunning Guidelines
+##################
+
+This guide provides a process through which the user can adjust the tunable parameters to obtain
```

Comment:
This is not (for now) so much a "guide" as it is a list of all parameters. Is the intention to provide a more fleshed out tuning guide here?

Reply:
Yes, I'm thinking this is a placeholder where we'll place links to tunning different components of the stack -- but for now, we only provide a list of the parameters.